doc/whats_new.rst

```diff
@@ -52,6 +52,8 @@ Changelog
 
     - Add the ability to read events when importing raw EEGLAB files, by `Jona Sassenhagen`_.
 
+    - Add function :func:`mne.viz.plot_sensors` for plotting sensor positions and :func:`mne.viz.plot_layout` for plotting layouts by `Jaakko Leppakangas`_
```

Comment:
Say that plot_sensors can also be called via data container such as mne.Epochs.plot_sensors and you should rather advertise the layout.plot method